prompting/validators/reward/reward.py

```diff
@@ -25,27 +25,17 @@
 
 @dataclass
 class BaseRewardEvent:
-    reward: float = None
-    normalized_reward: float = None
-    is_filter_model: bool = False
+    reward: float = 1.0
```

Comment:
Shall we keep reward default to 1 or None?

Reply:
If we keep to None the code breaks, that's why I re-implemented default == 1